### ISSUE TITLE
[Effect Suggestion] Lock Camera in Place & Fake Game Crash

### ISSUE THREAD

**Issue by pottkopp** [gta-chaos-mod/ChaosModV#335]
1. Lock Camera in Place - It happened 2 times for me already that the game bugged and locked the camera in place after respawning. Maybe it's worth an effect? So you could only use the map to progress.
2. Fake Game Crash - Maybe it's possible to hide or minimize the game for some seconds so it looks like it crashed?

**Comment by kolyaventuri** [comment]
Fake game crash is already an effect! Minimizing the game could be an improvement though, if it's possible to do

**Comment by pottkopp** [comment]
The Fake Crash already existing in the mod is a Fake Mod Crash. I might be alone with this opinion but to me those two are different. The one existing fakes the mod crashing and my suggestion fakes the whole game crashing.

**Comment by SeanMirrsen** [comment]
No, actually. The Fake Crash is faking the game crashing. The kind where it has some kind of unhandled exception and freezes momentarily before closing. You can see the many occasions of the game crashing during DarkViperAU's streams, and the Fake Crash is nearly indistinguishable from that, for the couple of seconds it lasts.

The crash you're thinking of is a much harder crash, where the game outright closes without warning, just straight CTD.

I'm not sure the CTD variant is worth faking, for a number of reasons, not least being that it's much harder to pull off, and switching focus may not be so painless for somebody running the game in exclusive fullscreen, for instance.

**Comment by kolyaventuri** [comment]
I can definitely look into minimizing / muting the game to make it seem like a real game crash

**Comment by SeanMirrsen** [comment]
I suppose it would be fine, since it'll be optional and if it causes anyone trouble they can turn it off. The trick would be seamlessly restoring focus again, especially if the user manages to click anywhere while the desktop is visible.

**Comment by pottkopp** [comment]
Maybe it's possible to change the graphics to borderless window with a resolution of 1x1 and after a few secs back to fullscreen?

**Comment by JustTunerYT** [comment]
If i remember right, after game minimizes, it pauses itself. But i think it's just a setting, that can be turned off to continue playing while minimized
